bedboss/bedboss.py

```diff
@@ -223,25 +324,33 @@ def main(test_args: dict = None) -> NoReturn:
 
     args_dict = vars(args)
 
+    pm_out_folder = (
+        args_dict.get("outfolder")
+        or args_dict.get("output_folder")
+        or "test_outfolder",
+    )
+    pm_out_folder = os.path.join(os.path.abspath(pm_out_folder[0]), "pipeline_manager")
```

Comment:
Why is `pm_out_folder` a list and not a string? It stands out to me. I was expecting to simply pass `pm_out_folder`?

Reply:
So if pm_out_folder is a str, won't taking an index at [0], just return the first char in the string? And cause this to error?

Reply:
we discussed, it is because it could return a tuple.